### ISSUE TITLE
Feature Request: Allow customization of font+spacing in User Feedback widget

### ISSUE THREAD

**Issue by willtsai** [google/docsy#679]
In the [configuration for the User Feedback Widget](https://www.docsy.dev/docs/adding-content/feedback/#setup-1), I would like to have the ability to customize the font size and spacing of the text. Attached is a screenshot example where the ability to reduce the font size and add a larger top margin for the widget text would allow for some separation between the core content on the page and the User Feedback widget near the page footer.

![image](https://user-images.githubusercontent.com/28876888/132073422-d4d5cdad-e08a-45a4-9820-844856a16612.png)

**Comment by willtsai** [comment]
PR to address this: https://github.com/dapr/docs/pull/1767

**Note:** on some pages there isn't sufficient space between the User Feedback widget and core website content, and I didn't find a way to configure the text size/spacing in the widget. I've created a [feature request](https://github.com/google/docsy/issues/679) to the Docsy team to allow us to fix this, but it might be something we have to live with for now. 

![image](https://user-images.githubusercontent.com/28876888/132074881-133dcd1e-9db5-4e1b-b322-a3b1f2c4e1a9.png)
![image](https://user-images.githubusercontent.com/28876888/132074920-8ff4f85b-3cfb-4dde-be9f-459e008c255b.png)

**Comment by narrenfrei** [comment]
I don't think, that a manually correction of the space is really the best way. In my opinion, we should try that the space between "real content" and the feedback widget is correct by default. 

Could you share links to a correct space and also to a page with too small space?

**Comment by narrenfrei** [comment]
I took a look on this page of your Dapr Docs:
https://docs.dapr.io/getting-started/

There is already too little space after the Button "First step: Install the Dapr CLI >>" now. So you have to increase the margin-bottom of this button and not the margin-top of the feedback widget. For example, by giving the button the class ```mb-4```.

**Comment by willtsai** [comment]
The tight margins on the buttons is something I can look into for sure (might actually be the intentional shortcoded button design). However, I still feel that in a lot of pages, the spacing in between the page content and Feedback wizard is too small. I think this is especially pronounced for pages that end with a shorter paragraph, like the one below.

I don't think that going through page by page to audit and add more spacing where necessary per page is the way I'd like to address this, so in my mind there are two options without needing to edit each page individually:

1. Have the ability to configure the margins/spacing for the Feedback wizard itself (I prefer this because then the extra margin space for core content will only be added when it's needed, i.e. when Feedback wizard is introduces)
2. Configure overall website page margins to add a little more spacing at the bottom margin to accommodate footer type content like Feedback wizard (need to investigate how to do this, but I prefer this less as I'd be modifying core content to accommodate Feedback wizard, which then needs to be remembered to be undone should someone choose to remove the Feedback wizard in the future)

Thanks for looking into this,
-Will

https://docs.dapr.io/contributing/contributing-overview/
![image](https://user-images.githubusercontent.com/28876888/132420914-dba71b37-3e62-42ad-b3c8-2d526ce54c47.png)

**Comment by narrenfrei** [comment]
If really a option for page individually more margin is needed, I'd prefer to add a parameter to add HTML classes to  the feedback widget or complete page. (If we don't have such an option already.) So styling can be done by CSS. 

But before I add such an option, I would like to get the OK from @LisaFC (or someone else from the "Core-Team"), wether my suggestion is in the sense of the project.

**Comment by LisaFC** [comment]
Hmm, interesting. I wonder is the default spacing between the main text and the Feedback Widget just too tight in general - I'm not sure if we thought about it that much when we added it - so might be something to just change as the default rather than make configurable (without making it so far down that users miss it without scrolling)? Or maybe add a line before it?

**Comment by willtsai** [comment]
I agree with changing it as the default rather than make it configurable. I would suggest adding **two** lines before it just to ensure that there is additional separation so as to not make the widget look like an additional paragraph and part of the core content.

**Comment by narrenfrei** [comment]
I would add margin-top: 2rem to the div of the feedback.

But as this div has only the class d-print-none, I would also add an class to the div like td-feedback.

I've also recognized that thee feedback has inline CSS, which should packed into the global stylesheet.

@LisaFC OK?

**Comment by chalin** [comment]
- Pre-0.17.0 triage campaign: #2717
- Consensus favored fixing the default spacing; no follow-up since 2021.
